src/OrderFormAcceptanceTests.Steps/Steps/CommencementDate.cs

```diff
@@ -0,0 +1,100 @@
+﻿using FluentAssertions;
+using OrderFormAcceptanceTests.Steps.Utils;
+using System;
+using TechTalk.SpecFlow;
+
+namespace OrderFormAcceptanceTests.Steps
+{
+	[Binding]
+    public class CommencementDate : TestBase
+    {
+		public CommencementDate(UITest test, ScenarioContext context) : base(test, context)
+		{
+		}
+
+        [Given(@"a valid date is entered")]
+        public void GivenAValidDateIsEntered()
+        {
+            var today = DateTime.Today;
+            GivenTheDayIsSetTo(today.Day.ToString());
+            GivenTheMonthIsSetTo(today.Month.ToString());
+            GivenTheYearIsSetTo(today.Year.ToString());
```

Comment:
is there any value in a method CommencementDate.SetDate(Date value) ?
This then inputs into each field under the hood
And you can use this in the two places your setting all the fields